pkg/plugin/plugin.go

```diff
@@ -84,10 +85,21 @@ func (j *JiraPlugin) Validate(ctx context.Context, req *jvspb.ValidateJustificat
 		}, fmt.Errorf("failed to validate justification: %w", err)
 	}
 
+	// There is only one JQL and one issueKey, so the first match result is
+	// checked directly.
+	if len(result.Matches[0].MatchedIssues) == 1 {
+		issueID := strconv.Itoa(result.Matches[0].MatchedIssues[0])
+		return &jvspb.ValidateJustificationResponse{
+			Valid:   true,
+			Warning: result.Matches[0].Errors,
+			Annotation: map[string]string{
+				"jira_issue_id": issueID,
```

Comment:
I thought we are going to add the URL here as well? Or you plan to do that separately